common-lib/src/main/scala/com/gu/mediaservice/lib/usage/ItemToMediaUsage.scala

```diff
@@ -0,0 +1,93 @@
+package com.gu.mediaservice.lib.usage
+
+import java.net.URI
+
+import com.amazonaws.services.dynamodbv2.document.Item
+import com.gu.mediaservice.model.usage._
+import org.joda.time.DateTime
+import org.joda.time.format.ISODateTimeFormat
+
+import scala.collection.JavaConverters._
+import scala.util.Try
+
+object ItemToMediaUsage {
+
+  def transform(item: Item) = {
```

Comment:
Can this have a response type please given it's a public function.

Reply:
sure

Reply:
done